crawler4j-commons/src/main/java/edu/uci/ics/crawler4j/crawler/DynamicCrawlConfig.java

```diff
@@ -0,0 +1,101 @@
+package edu.uci.ics.crawler4j.crawler;
+
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.util.Collections;
+import java.util.List;
+
+import static edu.uci.ics.crawler4j.crawler.DynamicCrawlConfig.WebDriverType.firefox;
+
+/**
+ * Extends {@link CrawlConfig} to add configuration properties for the Selenium's <em>WebDriver</em>.
+ */
+public class DynamicCrawlConfig extends CrawlConfig {
+
+    public enum WebDriverType {
+        chrome,
+        firefox
+    }
+
+    private WebDriverType webDriverType = firefox;
+
+    /**
+     * If dynamic content crawling is used, the maximum time to wait for dynamic content loading in seconds.
+     */
+    private int maxWaitForDynamicContentInSeconds = 2;
+
+    /**
+     * Absolute path to the driver binary on the local filesystem.
+     */
+    private String webDriverPath;
+
+    /**
+     * The user agent to be used by the web driver.
+     */
+    private String webDriverUserAgent;
```

Comment:
Is there a reason for not using the value from https://github.com/rzo1/crawler4j/blob/master/crawler4j-commons/src/main/java/edu/uci/ics/crawler4j/crawler/CrawlConfig.java#L65 ? Do we have any good reasons for introducing a 2nd user agent config option?

Reply:
Hmmm actually no, you're right. I'll remove it.